source/server-discovery-and-monitoring/server-discovery-and-monitoring.md

```diff
@@ -485,7 +487,14 @@ removed once the primary is checked.
 #### error
 
 If the client experiences any error when checking a server, it stores error information in the ServerDescription's error
-field.
+field. The message contained in this field MUST contain the substrings detailed in the table below when the
+ServerDescription is changed to Unknown in the circumstances outlined.
+
+| circumstance                                                         | error substring                                          |
+| -------------------------------------------------------------------- | -------------------------------------------------------- |
+| RSPrimary with a stale electionId is discovered                      | `'primary marked stale due to electionId mismatch'`      |
+| RSPrimary with a stale setVersion is discovered                      | `'primary marked stale due to setVersion mismatch'`      |
```

Comment:
Can we combine these two cases and include both the stale and max election tuples?

Reply:
Yeah, that's fair